### PR TITLE
css-canvas-medical - James Ding

### PR DESCRIPTION
**Pull request by jamessding**
Screen Shot 2022-05-14 at 5 53 47 PM
Screen Shot 2022-05-14 at 5 54 11 PM
Screen Shot 2022-05-14 at 5 54 42 PM

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey James, it looks like your button in the "Ready to learn more?" section is misaligned, please revise and submit a new screenshot with your revision.

**Comment by jamessding** [comment]
Screen Shot 2022-05-14 at 5 54 42 PM

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
That button is meant to be centered vertically in that section, but it looks like yours is still resting towards the top of the section

File: css-canvas-medical/index.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+      </div>
+      <hr>
+    </div>
+    <div class="row">
+      <div class="column-one-third color-grey text-align-left padding-left padding-bottom-30">
+        <h2 class="font-tiempos">Ready to learn more?</h2>
+        <p class="font-avenir font-size-22">Schedule a private meeting now</p>
+      </div>
+      <div class="column-one-third">
+      </div>
```
Comment (codingcodymiller) [inline css-canvas-medical/index.html:68]: Uh oh, this is bad news. You should never use empty columns to create space. Instead, you can make the previous element `column-two-thirds` or make use of a utility class with `justify-content` to control where the items in this row are aligned.

File: css-canvas-medical/index.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        <img class="logo padding-bottom-30" src="images/logo-white.png">
+        <div class="contact">
+          <p class="margin-5">2307 Irving Street, Suite 228</p>
+          <p class="margin-5">San Francisco CA 94122</p>
+          <p class="margin-5">(800) 370-1416</p>
+          <p class="margin-5">info@canvasmedical.com</p>
+        </div>
+      </div>
+      <div class="column-one-fourth">
+      </div>
```
Comment (codingcodymiller) [inline css-canvas-medical/index.html:84]: Same problem here.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]
Nice work on this across the board, good revisions!!

**Comment by jamessding** [comment]
Screen Shot 2022-05-27 at 5 31 46 PM
Screen Shot 2022-05-27 at 5 32 27 PM
Screen Shot 2022-05-27 at 5 32 40 PM
Screen Shot 2022-05-27 at 5 32 52 PM
Screen Shot 2022-05-27 at 5 32 58 PM

**Pull request merged**